components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ManagementInternalAPI.java

```diff
@@ -35,6 +37,12 @@
 
     private String name;
 
+    private static Log log = LogFactory.getLog(ManagementInternalAPI.class);
```

Comment:
This constant name does not match the WSO2 code style. Shall we fix?

Reply:
Fixed in 2fcc0a2